deploy/helm/sumologic/templates/setup/setup-clusterrole.yaml

```diff
@@ -3,10 +3,10 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   name:  {{ template "sumologic.fullname" . }}-setup
+  {{- if .Values.sumologic.setup.clusterRole.annotations  }}
   annotations:
-    "helm.sh/hook": pre-install
-    "helm.sh/hook-weight": "1"
-    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+{{ toYaml .Values.sumologic.setup.clusterRole.annotations | indent 4 }}
```

Comment:
does this work the same without the `if` conditional? That is, if `.Values.sumologic.setup.clusterRole.annotations` doesn't exist, it simply won't add it?

Could we skip the `{{- if ... }}` and `{{- end }}` in all of these?

Reply:
oh, true